Common/Algorithm/Framework/Portfolio/PortfolioTarget.cs

```diff
@@ -67,6 +74,19 @@ public static IPortfolioTarget Percent(IAlgorithm algorithm, Symbol symbol, doub
             return Percent(algorithm, symbol, percent.SafeDecimalCast());
         }
 
+        /// <summary>
+        /// Creates a new target for the specified percent
+        /// </summary>
+        /// <param name="algorithm">The algorithm instance, used for getting total portfolio value and current security price</param>
+        /// <param name="symbol">The symbol the target is for</param>
+        /// <param name="percent">The requested target percent of total portfolio value</param>
+        /// <param name="tag">The target tag with additional information</param>
+        /// <returns>A portfolio target for the specified symbol/percent</returns>
+        public static IPortfolioTarget Percent(IAlgorithm algorithm, Symbol symbol, double percent, string tag)
```

Comment:
Wondering if we can directly just add `, string tag = ""` to the constructor above like in the other cases?